pkgs/development/python-modules/authres/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchurl, openssl, makeWrapper, buildPythonApplication
+, pytest, dnspython, pynacl, authres }:
+
+buildPythonApplication rec {
+  pname = "authres";
+  majorversion = "1.0";
+  minorversion = "1";
+  version = "${majorversion}.${minorversion}";
+
+  src = fetchurl {
+    url = "https://launchpad.net/authentication-results-python/${majorversion}/${majorversion}.${minorversion}/+download/${pname}-${version}.tar.gz";
+    sha256 = "1crdff9wa4cma6p4a6ldiffa6c3awxafnv4d207h6z6msc3jb868";
+  };
+
+  buildInputs = [ pytest ];
```

Comment:
`checkInputs`?

Reply:
I had not come across that, but that is completely logical.